charts/prometheus-blackbox-exporter/values.yaml

```diff
@@ -185,6 +185,10 @@ service:
   labels: {}
   type: ClusterIP
   port: 9115
+  ipDualStack:
+    enabled: false
+    ipFamilies: ["IPv6", "IPv4"]
```

Comment:
That would fail on a cluster with IPv4 being the first IP Family aka default. I hope that's understood.